### ISSUE TITLE
install libsbml on mac os using Matlab interface

### ISSUE THREAD

**Issue by krithikakrishnan04** [sbmlteam/libsbml#279]
Hi,

I am new to Matlab.  
I am using LibSBML 5.16.0 in matlab 2020a. I'm on macOS Monterey 12.6
I downloaded libsbml through the matlab interface (download link below), https://sourceforge.net/projects/sbml/files/libsbml/MATLAB%20Interface/libSBML-5.18.0-matlab-oldmacosx-binaries.tar.gz/download
After adding the path to matlab, I tested TranslateSBML('test.xml') and i got the following error:

`Attempt to execute SCRIPT TranslateSBML as a function:
/Users/krithika/Downloads/libSBML-5.19.0-matlab-binaries/TranslateSBML.m`

I tried option 2 as well: 

> Extract the archive somewhere on your local computer.
Start your copy of MATLAB, navigate to the extracted folder and run 'installSBML'. This will add the path and verify that everything works (as above).

It threw me an error: 

`run /Users/krithika/Downloads/libSBML-5.19.0-matlab-binaries/installSBML.m
Installing the libSBML interface. * Doing preliminary checks of runtime environment ...
  This appears to be MATLAB and not Octave.
Checking for executables in 
	/Users/krithika/Downloads/libSBML-5.19.0-matlab-binaries
Executables not found
Error using installSBML (line 58)
Executables were not found.Please make sure they are in the same directory as
the installSBML and other scripts.
Error in run (line 91)
evalin('caller', strcat(script, ';'));`

Thanks

**Comment by fbergmann** [comment]
I currently dont have access to Matlab on macOS. For some reason, it seems that matlab is not loading the compiled binary (`TranslateSBML.mexmaci64`). I'm unsure as to why that is. could you run in the Terminal: 

```bash 
cd /Users/krithika/Downloads/libSBML-5.19.0-matlab-binaries/
otool -L TranslateSBML.mexmaci64 
```

**Comment by krithikakrishnan04** [comment]
I ran it. Please see the screenshot attached. ![image](https://user-images.githubusercontent.com/7892259/200879840-1963b30a-bb38-4851-bd62-eb6b4cc1d4c1.png)

**Comment by fbergmann** [comment]
This is really strange, when i download the zip file, i have a `TranslateSBML.mexmaci64` file in that directory. I'm not sure where it is gone. Could you just download: 

```
https://master.dl.sourceforge.net/project/sbml/libsbml/5.19.0/stable/MATLAB%20interface/libSBML-5.19.0-matlab-binaries.zip?viasf=1
```

once more, and extract it, and verify that it is present. There really is no need for any installation procedure, other than adding the folder to the matlab path. 

Thanks

**Comment by krithikakrishnan04** [comment]
So the file, "TranslateSBML.mexmaci64" was not available in my previous download of "libSBML-5.19.0-matlab-binaries" which was a tar.gz file. I unzipped it again and now the file is present. I discovered this file in your zipped folder link too. I was able to run test.xml now. Thank you so much!

**Comment by skeating** [comment]
Sorted
